### ISSUE TITLE
Cannot load extension/plugin feature due to software dependencies

### ISSUE THREAD

**Issue by B-Hartmann** [DC-analysis/DCscope#130]
Windows 10
Shape-Out 2.11.5 (installed via installer)

I followed the instructions in the docs https://shapeout2.readthedocs.io/en/stable/sec_qg_extensions.html
to add a custom plugin feature and got an error message saying that I "might have to install additional software".

If a plugin feature requires a package that was not installed by the installer of Shape-Out, how can make the code run?
Do I have to install Shape-Out via pip?

**Comment by paulmueller** [comment]
Yes, you would have to install Shape-Out via pip. There is no clear path on how to proceed with this issue in the future. One solution might be to have two Shape-Out installers: One minimal and one with multiple additional Python packages (including tensorflow, pytorch, opencv, etc...).
